lib/flight_job/outputs/info_job.rb

```diff
@@ -28,107 +28,99 @@
 require 'output_mode'
 
 module FlightJob
-  module Outputs::InfoJob
-    extend OutputMode::TLDR::Show
-
-    TEMPLATE = <<~ERB
-      <% verbose = output.context[:verbose] -%>
-      <% each(:default) do |value, padding:, field:| -%>
-      <%
-          # Apply the colours
-          value = pastel.green value
-          field = pastel.blue.bold field
-      -%>
-      <%= padding -%><%= pastel.blue.bold field -%><%= pastel.bold ':' -%> <%= value %>
-      <% end -%>
-      <%
-        submit_outputs = output.procs.select { |proc| proc.config[:section] == :submit }
-        if verbose || output.context[:submit]
-      -%>
-
-      <%= pastel.blue.bold "Submit Stdout" -%><%= pastel.bold ':' %>
-      <%= pastel.green submit_outputs.first.call(model) %>
-      <%= pastel.blue.bold "Submit Stderr" -%><%= pastel.bold ':' %>
-      <%= pastel.green submit_outputs.last.call(model) %>
-      <% end -%>
-    ERB
-
-    register_attribute(header: 'ID') { |j| j.id }
-    register_attribute(header: 'Script ID') { |j| j.script_id }
-    register_attribute(header: 'Scheduler ID') { |j| j.scheduler_id }
-    register_attribute(header: 'State') { |j| j.state }
-
-    # Show a boolean in the "simplified" output, and the exit code in the verbose
-    # NOTE: There is a rendering issue of integers into the TSV output. Needs investigation
-    register_attribute(header: 'Submitted', verbose: false) { |j| j.submit_status == 0 }
-    register_attribute(header: 'Submit Status', verbose: true) { |j| j.submit_status.to_s }
-
-    register_attribute(header: 'Submitted at') do |job, verbose:|
-      if verbose
-        job.created_at
-      else
-        DateTime.rfc3339(job.created_at).strftime('%d/%m/%y %H:%M')
-      end
+  class Outputs::InfoJob < OutputMode::Formatters::Show
+    def initialize(job, submit: true, **opts)
```

Comment:
The stdout and stderr is being included always.  I suspect its due to `submit: true` being the default here.